src/resultscallercalleepage.cpp

```diff
@@ -93,9 +93,9 @@ ResultsCallerCalleePage::ResultsCallerCalleePage(FilterAndZoomStack* filterStack
     ResultsUtil::setupHeaderView(ui->callerCalleeTableView, contextMenu);
     ResultsUtil::setupCostDelegate(m_callerCalleeCostModel, ui->callerCalleeTableView);
     connect(ui->callerCalleeTableView, &QTreeView::activated, this, [this](const QModelIndex& index) {
-        auto symbol = index.data(CallerCalleeModel::SymbolRole).value<Data::Symbol>();
-        if (symbol.isValid())
-            emit jumpToDisassembly(symbol);
+        m_symbol = index.data(CallerCalleeModel::SymbolRole).value<Data::Symbol>();
```

Comment:
`activated` gets emitted on double click on my system - it's imo meaningless to set some member variable then. instead, the context menu should  check the current index and ask it for its symbol